src/system/sys_profiler.hh

```diff
@@ -63,6 +63,10 @@
 #include "global_defs.h"
 #include "system/system.hh"
 #include <mpi.h>
+#include <boost/property_tree/ptree.hpp>
+#include "time_point.hh"
+
+using boost::property_tree::ptree;
```

Comment:
Use just namespace rename:
e.g. namespace ptree=boost::property_tree::ptree

This improves readability since reader notice that some class or function is related to some library.